setup.py

```diff
@@ -125,7 +125,7 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=[s.strip() for s in metadata.get('install_requires', 'astropy').split(',')],
+      install_requires=[s.strip() for s in metadata.get('install_requires', 'astropy').strip().split(',')],
```

Comment:
You tell me. That same `.strip()` was in the original setup.py file, so I'm just replicating it.